web/templates/web/demographic-data-update.html

```diff
@@ -12,25 +12,89 @@
 {% block content %}
     <script>
         $(document).ready(function() {
-            if ($('#id_country')[0].value !== 'US') {
-                hideField('state');
-            }
+        
+			const gdpr_country_list = [	
+				"", // don't accept demographic data with NO country listed
+				"AT", 
+				"BE", 
+				"BG", 
+				"CY", 
+				"CZ", 
+				"DE", 
+				"DK", 
+				"EE", 
+				"ES", 
+				"FI", 
+				"FR", 
+				"GB", 
+				"GR", 
+				"HR", 
+				"HU", 
+				"IE", 
+				"IT", 
+				"LT", 
+				"LU", 
+				"LV", 
+				"MT", 
+				"NL", 
+				"PL", 
+				"PT", 
+				"RO", 
+				"SE", 
+				"SI", 
+				"SK",
+				"NO", // protections also apply to Norway, Iceland, & Liechtenstein
+				"IS",
+				"LI"];
+			
+			const gdprFieldsToHide = ['number_of_children', 'child_birthdays', 
+            		'languages_spoken_at_home', 'number_of_guardians', 
+            		'number_of_guardians_explanation', 'race_identification', 
+            		'age', 'gender', 'education_level', 'spouse_education_level',
+            		'annual_income', 'number_of_books', 'density'];
+        
             if ($('#id_number_of_children')[0].value === "0" || $('#id_number_of_children')[0].value === '') {
                 hideField('child_birthdays');
                 $('.help-block').first().hide();
             }
             if ($('#id_number_of_guardians')[0].value !== 'varies') {
                 hideField('number_of_guardians_explanation');
             }
-
+            
+            if ($('#id_country')[0].value !== 'US') {
+                hideField('state');
+            }
+            
+            if (gdpr_country_list.includes($('#id_country')[0].value)) {
+            	$.each(gdprFieldsToHide, function(i, fieldName) {
+            		hideField(fieldName);
+            		$(`#id_${fieldName}`)[0].value = '';
+            		$(`#id_${fieldName} input[type=checkbox]`).prop("checked", false );
+            	});
+            }
+            
             $('#id_country').on('change', function() {
+            	// First show/hide state field based on USA or not
                 if ($(this)[0].value === 'US') {
                     showField('state');
                 } else {
                     hideField('state');
                     $(`#id_state`)[0].value = '';
                 }
+                // Then show/hide most demographic fields based on EU or not
+				if (gdpr_country_list.includes($('#id_country')[0].value)) {
+					$.each(gdprFieldsToHide, function(i, fieldName) {
+						hideField(fieldName);
+						$(`#id_${fieldName}`)[0].value = '';
+						$(`#id_${fieldName} input[type=checkbox]`).prop("checked", false );
```

Comment:
Minor: I would consolidate these three lines into a function that can be called both here and [here](https://github.com/lookit/lookit-api/pull/98/files#diff-b8d298b6a4a06a533d94e07abb55422cR70)

Reply:
Fixed, thanks!